client/spawn.lua

```diff
@@ -16,6 +16,18 @@ RegisterNUICallback('ox:setCharacter', function(data, cb)
 	end
 end)
 
+RegisterNUICallback('loadLocale', function(_, cb)
+    cb(1)
+    local resource = GetCurrentResourceName()
+    local locale = GetExternalKvpString('ox_lib', 'locale') or 'en'
```

Comment:
Locales are now loaded based on the `ox:locale` convar, not the kvp string.

Reply:
It should be fixed :D